content/en/docs/tutorials/stateless-application/guestbook.md

```diff
@@ -324,6 +324,10 @@ Scaling up or down is easy because your servers are defined as a Service that us
 {{% /capture %}}
 
 {{% capture cleanup %}}
+{{< note >}}
+The next tutorial [Example: Add logging and metrics to the PHP / Redis Guestbook example](../guestbook-logs-metrics-with-elk/) builds upon the infrastructure which you have created in this tutorial. If you plan to continue to that tutorial, you may want to skip the cleanup for now. The Cleanup at the end of that tutorial includes the cleanup steps from this tutorial.
```

Comment:
Hi @chakatz , I am not convinced that this note is needed.
That said, what do you think about simplifying the note, something like:
The [Example: Add logging and metrics to the PHP / Redis Guestbook example](/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk/) tutorial builds upon the resources created in this tutorial. If you plan to continue with the "logging and metrics" tutorial, you may want to wait to cleanup the guestbook resources.

Reply:
Hi @kbhawkey, That sounds good. More concise. Will you submit that rewrite or should I?

Regarding the need, you can see at the link below that in that next tutorial it does reference retaining the build so it can be used, it's just that by the time people read that comment it's probably too late, out-of-luck, that's what happened to me.
https://kubernetes.io/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk/#start-up-the-php-guestbook-with-redis

Reply:
If you are able, you should make the edits and rebase your changes.

Reply:
https://kubernetes.io/docs/contribute/new-content/new-content/#merge-conflicts-and-rebasing